### DISCUSSION TITLE
Can't create an issue

### DISCUSSION THREAD

**Discussion by captenmasin** [diegoazh/gmap-vue#183] [General]
I can't create an actual issue because the bot keeps closing for not using the template, despite me carefully copying and pasting the template... anyways

**Describe the bug**
A weird issue in which maps don't render in CSS Columns

**To Reproduce**
1. Create 4 map embeds
2. Wrap them in a class
3. Add `columns: 4` to the wrapper class

**Expected behavior**
Expected 4 maps to display and pan correctly side-by-side as 4 columns

**Current behavior**
The first map displays correctly (but with a weird overflow issue, see video 2) but the rest don't seem to render

**Screenshots**
https://imgur.com/a/wRnlV93

**Desktop (please complete the following information):**
 - OS: MacOS Big Sur 11.4
 - Browser: Chrome
 - Version: 91.0.4472.114 (Official Build) (arm64)

**Smartphone (please complete the following information):**

**Additional context**

## Versions

- Node: v15.8.0
- NPM: 7.5.4

## Package manager

- [x] NPM
- [ ] Yarn

## Plugin version

- [x] 1.5.x
- [ ] 1.4.x
- [ ] 1.2.x
- [ ] 1.0.x

**Comment by diegoazh** [comment]
I'm checking that, sorry the delay but I was working on the new documentation and the new plugin structure

**Comment by diegoazh** [comment]
@captenmasin I think I fix the bot, I test it and works well for me but I'm not completely sure, feel free to open a new one only for testing purposes, if it works I will delete it later.
Thank you for opening the first discussion here, and one more time apologizing for the mess that causes the bot.
